app/code/Magento/ProductAlert/Model/Observer.php

```diff
@@ -45,112 +64,121 @@ class Observer
      *
      * @var array
      */
-    protected $_websites;
+    private $websites;
```

Comment:
This is legacy class, we need to follow backward compatibility
So need to revert all changes in this class
We change the only `if ($this->productIsSalable->isSalable` condition